### DISCUSSION TITLE
Github public repo is not publishing, giving 404 error

### DISCUSSION THREAD

**Discussion by sweetarebero** [community/community#102947] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I have tried evrything but github pages is giving me 404

**Comment by DaviZCodes** [comment]
I saw you were trying to host a React Page to GitHub pages. Have you installed gh-pages onto your app and added deployment scripts?

This link is very helpful: https://github.com/gitname/react-gh-pages

Also, check if you are on the right branch.

**Reply by sweetarebero** [reply]
Hi Davi, yes I have done all the above. My page was actually working. But i changed the visibility to Private then realized on free version it doesn't work. So i changed visibility back to Public now and even now it shows 404 :(

**Reply by DaviZCodes** [reply]
Oh I see it stopped working after you changed it.

Perhaps try removing your deployment completely - delete all deployments and start from scratch. If that doesn't work, create a new repository and clone this one onto the other one and try deploying it. 

Perhaps GitHub still thinks your repo is private, so it is bugging.
